### PR TITLE
Feature/uw 65

### PR DESCRIPTION
**Pull request by jprestop**
(I copied in the pull request template we use for the METplus components - if it's too much, I can be sure not to use it in the future.  I just wanted to provide the structure I am used to for this pull request, but understand if others don't want it.)

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>

I did not do any testing. Testing will be done once ReadTheDocs is set up and GitHub Actions runs using the modified docs.yaml file in .github/workflows.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review the files changed and ensure the changes seem reasonable.  

I did add some documentation taken from this [document](https://docs.google.com/document/d/1jWdRmvPv0lgWlxMrr1lGY4W4KQ83_moZU_EGQslVtqM/edit#).  Please make any suggestions for modifications or deletions.  I mainly just wanted to have some basic content in place.
 
- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes or No]**

I am uncertain if there will be any errors or warnings.  I added actions so that a warning/error log would be produced as an artifact, but I'm not sure if this has to be merged in before that will take effect or not. We may need to do more work in this branch if there are problems with this initial set up.

- [x] Do these changes include sufficient testing updates? **[No]**

See comments in the above sections regarding testing.

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>
I don't believe we have any unit or regression testing set up at this point, but any tests should not change existing output.

- [x] Please complete this pull request review by **[As soon as you're able to]**.</br>

### CODE REVIEW

**Review by christinaholtNOAA — DISMISSED** [review]
@jprestop This looks great. Thanks for helping us get all this set up!

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+###############
+United Workflow
```
Comment (aerorahul) [inline docs/index.rst:2]:
```suggestion
Unified Workflow
```

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+deterministic forecast to an ensemble of forecasts to cycled data assimilation
+systems.  The application suites are not just limited to a model and data
+assimilation system.  There are numerous examples of “workflows” in other
+applications, such as observation processing and ingest, post-processing of
+model and data assimilation generated data, reanalysis, verification and
+validation, deployment of products and graphics to web servers.
+
+A schematic of the vision above is shown in Figure 1 below.
+
+.. _Schematic_Layout_United_Workflow_Framework:
```
Comment (aerorahul) [inline docs/index.rst:28]:
```suggestion
.. _Schematic_Layout_Unified_Workflow_Framework:
```

File: docs/index.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	
+.. figure:: figure/Schematic_Layout_United_Workflow_Framework.png
```
Comment (aerorahul) [inline docs/index.rst:30]:
```suggestion
.. figure:: figure/Schematic_Layout_Unified_Workflow_Framework.png
```
Also, please change the name of the file corresspondingly.

File: docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+   *Figure 1: Schematic layout of the various components of the United Workflow Framework.*
```
Comment (aerorahul) [inline docs/index.rst:32]:
```suggestion
   *Figure 1: Schematic layout of the various components of the Unified Workflow Framework.*
```

File: docs/conf.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+# documentation root, use os.path.abspath to make it absolute, like shown here.                                                                                            
+#                                                                                                                                                                          
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
+print(sys.path)
+
+# -- Project information -----------------------------------------------------                                                                                             
+
+project = 'United Workflow'
```
Comment (aerorahul) [inline docs/conf.py:20]:
```suggestion
project = 'Unified Workflow'
```

File: docs/index.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Using the tools on the right, any application can assemble a unique suite for
+its needs.   This allows all applications to retain similar implementations
+while at the same time providing a mechanism to construct their own flavor of
+the implementation. 
+
+
+.. toctree::
+   :hidden:
+   :caption: United Workflow
```
Comment (aerorahul) [inline docs/index.rst:48]:
```suggestion
   :caption: Unified Workflow
```

File: docs/Users_Guide/index.rst [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+with other applications.
+
+It is important to note here that the Unified Workflow is an evolving software
+package. This documentation describes the |release|
+release dated |release_date|.
+
+
+**Unified Workflow TERMS OF USE - IMPORTANT!**
+
+Copyright |copyright|
```
[the remark this answers is not in the document]
Comment (aerorahul) [reply docs/Users_Guide/index.rst:24]: Yes. And we should.
The project is under the LGPL license (similar to UFS-weather-model and other NOAA applications).
A **Disclaimer** should be added.  The text of the disclaimer can also follow from the UFS-weather-model [here](https://github.com/ufs-community/ufs-weather-model#disclaimer)

File: .github/workflows/docs.yaml [hunk 8]
```diff
@@ -1,33 +1,51 @@
-name: Build and Deploy Documentation
+name: Documentation
```
Comment (aerorahul) [inline .github/workflows/docs.yaml:1]: How and where can we view this built documentation?
Is work required at the RTD website to setup hooks to pull the artifact?

File: docs/conf.py [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+author = 'NOAA/EMC, NOAA/GSL, GMU, CIRES, and UCAR/NCAR'
+author_list = 'Mahajan, R., C. Holt, B. Cash, F. Gabelmann, O. Adejumo, and J. Prestopnik'
+version = '0.0.0'
+verinfo = version
+release = f'{version}'
+release_year = '2022'
+release_date = f'{release_year}-03-31'
```
Comment (aerorahul) [inline docs/conf.py:27]: Can we have a single `release_date`
e.g.
```
release_date = '2022-03-31'
```
and deduce `release_year`, `release_month`, and `release_day` from it?

**Review by aerorahul — COMMENTED** [review]
This looks great.
Thank you for taking the time to set this up for the project.
I have a few comments:
1. `United` should be `Unified`.  I have tried to capture as many as I can.  Please search and replace all instances of `United Workflow` to `Unified Workflow`.
2. The documentation build looks good.  How does one deploy this to RTD site?  Is there work required at that end to look for triggers to grab the uploaded artifacts?

File: docs/Users_Guide/index.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (jprestop) [reply docs/Users_Guide/index.rst:24]: @aerorahul, thank you for the link to the Disclaimer.  i modified the from "**Unified Workflow TERMS OF USE - IMPORTANT!**" to be "**Disclaimer**" and added the text that you linked to.

File: docs/conf.py (hunk 9 above)
Comment (jprestop) [reply docs/conf.py]: Yes, and I just modified the file to do that.

File: .github/workflows/docs.yaml (hunk 8 above)
Comment (jprestop) [reply .github/workflows/docs.yaml:1]: @aerorahul You can partially view it in GitHub.  For example, see this [link](https://github.com/jprestop/workflow-tools/blob/feature/UW-65/docs/index.rst).  However, I know that is not really what you want to see.  I'll reply to your review comment below with more information on this.

**Comment by jprestop** [comment]
> This looks great.
Thank you for taking the time to set this up for the project.
I have a few comments:
United should be Unified. I have tried to capture as many as I can. Please search and replace all instances of United Workflow to Unified Workflow.
The documentation build looks good. How does one deploy this to RTD site? Is there work required at that end to look for triggers to grab the uploaded artifacts?

@aerorahul, Great catch on the "United" vs. "Unified" error.  I noticed I was doing that and thought I had used find/grep to correct other instances, but I looked back and found an error in my find command (searching for files with an incorrect extension).  Thank you for noticing the mistake!

I started setting up the RTD site, however, as I posted in Slack:

> I will need to have the pull request approved before setting up the ReadTheDocs pages with the webhook, so that the infrastructure will be in place to build in ReadTheDocs.  At that point, having admin privileges in the GitHub Repo (even if only temporarily) would be very helpful for me to be able to set up RTD more quickly.
Also, [@Fredrick](https://epic-slack-chan.slack.com/team/U02HW3LJCKS), [@Christina Holt - NOAA Affiliate](https://epic-slack-chan.slack.com/team/U02L74FH4G3), [@Rahul Mahajan](https://epic-slack-chan.slack.com/team/U02E59EME6T), [@Ope Adejumo](https://epic-slack-chan.slack.com/team/U02FXHYRAMQ) and [@Ben Cash](https://epic-slack-chan.slack.com/team/U02EL19AZ8B), if you do not already have a ReadTheDocs account, please create one [here](https://readthedocs.org/accounts/signup/) (many of us have used the “Sign up with GitHub” button) and let me know what your username is so that I can list you as a “Maintainer” of the RTD pages.  If you have already set up an account on RTD, please let me know what your username is.

In previous meetings, we had talked about giving some other folks admin access to the repo.  If you're able to provide me with admin access to the repo, even if only temporarily, it would be helpful for me to finish setting up RTD.  Otherwise, I would likely need to do a screen sharing session with an admin of this repo so that I can compare the configuration of this repo to the METplus repo and set up the webhook and test. 

@aerorahul @christinaholtNOAA @ope-adejumo @benjamin-cash @fgabelmannjr, Just a reminder to send your ReadTheDocs username once you have one so that I can add you as a maintainer.  I'll also follow up in Slack about this as some names don't appear to be pulling up here, and I'm not sure that you'll get notification of this message or not.

**Pull request merged**